cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaRebalanceAssemblyOperator.java

```diff
@@ -387,15 +387,15 @@ private String rebalanceStateConditionType(KafkaRebalanceStatus status) {
                                                                         KafkaRebalanceAnnotation rebalanceAnnotation, RebalanceOptions.RebalanceOptionsBuilder rebalanceOptionsBuilder) {
         switch (currentState) {
             case New:
-                return onNew(reconciliation, host, apiClient, rebalanceOptionsBuilder);
+                return onNew(kafkaRebalance, reconciliation, host, apiClient, rebalanceOptionsBuilder);
```

Comment:
The reconciliation should go first, you should place the `kafkaRebalance` in 4th position as it is in the other methods. Your IDE should allow you to refactor the method signature automatically.

Reply:
Okay, I will do it